app/cdap/components/ColumnInsights/Components/ColumnDetails/index.tsx

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, IconButton } from '@material-ui/core';
+import { grey, red } from '@material-ui/core/colors';
+import EditIcon from '@material-ui/icons/Edit';
+import RenderLabel from 'components/ColumnInsights/Components/common/RenderLabel';
+import InputSelect from 'components/ColumnInsights/Components/InputSelect';
+import { DATATYPE_OPTIONS } from 'components/ColumnInsights/options';
+import T from 'i18n-react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const PREFIX = 'features.WranglerNewUI.ColumnInsights';
+interface IColumnDetailsProps {
+  columnName: string;
+  characterCount: string;
+  distinctValues: number;
+  dataTypeString: string;
+  renameColumnNameHandler: (oldColumnName: string, newColumnName: string) => void;
+  dataTypeHandler: (dataType: string) => void;
+  columnType: string;
+  columnHeaderList: string[];
+}
+
+const ColumnDetailsContainer = styled(Box)`
+  padding-bottom: 20px;
+  border-bottom: 1px solid ${grey[300]};
+`;
+
+const CustomizedColumnNameEditBox = styled(Box)`
+  display: flex;
+  gap: 16px;
+`;
+
+const CustomizedIconButton = styled(IconButton)`
+  cursor: pointer;
+  justify-content: flex-start !important;
+`;
+
+const ColumnInsightsDetailsBox = styled(Box)`
+  margin-top: 20px;
+`;
+
+const ColumnInsightsDetailsCountBox = styled(Box)`
+  display: flex;
+  gap: 27px;
+  align-items: center;
+  margin-bottom: 7px;
+`;
+
+export default function({
+  columnName,
+  characterCount,
+  distinctValues,
+  dataTypeString,
+  renameColumnNameHandler,
+  dataTypeHandler,
```

Comment:
Please name handlers using an `onSomething` pattern. `onColumnNameChange`, `onColumnDataTypeChange`

Reply:
Changed it to onColumnNameChange and onColumnDataTypeChange